src/assets/css/multiple-authors-widget.css

```diff
@@ -9,10 +9,10 @@
     position: relative;
     min-height: 35px;
     line-height: 35px;
-    border-bottom: 1px solid #dddddd;
-    border-top: 1px solid #dddddd;
-    border-bottom: 1px solid #dddddd;
-    border-top: 1px solid #dddddd;
+    border-bottom: 1px solid #999;

```

Comment:
Since the Twenty Twenty-One theme  uses green as background, the border color is not visible.